### PR TITLE
Add cron jobs.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@pmclanahan r?

File: scripts/cron.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            self.log('CRASHED: {}'.format(e))
+            raise
+        else:
+            self.log('finished successfully')
+
+    def log(self, message):
+        print('Clock job {0}: {1}'.format(self.name, message))
+
+
+def ping_dms(function):
```
Comment (pmclanahan) [inline scripts/cron.py]: Nice :+1:

**Comment by pmclanahan** [comment]
Nice work! r+ :fireworks:

**Comment by glogiotatidis** [comment]
woohoo!

**Pull request merged**